### PR TITLE
LRN coding style changes

### PR DESCRIPTION
**Pull request by pengzhao-intel**
## Description ##
LRN coding style changes

### Essentials ###
- [ ] Passed code style checking (`make lint`)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage
- [ ] For user-facing API changes, API doc string has been updated. For new C++ functions in header files, their functionalities and arguments are well-documented. 
- [ ] To my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

File: src/operator/nn/mkldnn/mkldnn_lrn-inl.h [hunk 1]
```diff
@@ -97,17 +100,17 @@ void MKLDNNLRN_Forward(const OpContext &ctx, const LRNParam &param,
...
+                        const OpReqType req,
+                        const NDArray &in_grad) {
   if (req == kNullOp) {
     return;
   }
   // Repeat FW for getting workspace
-  auto data_mem = in_data.GetMKLDNNData();
-  auto data_md = data_mem->get_primitive_desc().desc();
-  auto pdesc_fwd = GetLRNFwd(param, ctx.is_train, data_md);
+  const auto data_mem = in_data.GetMKLDNNData();
```
Comment (zheng-da) [inline src/operator/nn/mkldnn/mkldnn_lrn-inl.h]: i think GetMKLDNNData returns const mkldnn::memory *

File: src/operator/nn/lrn.cc [hunk 2]
```diff
@@ -122,7 +125,7 @@ void LRNComputeExCPU(const nnvm::NodeAttrs &attrs,
                      const std::vector<NDArray> &outputs) {
   const LRNParam &param = nnvm::get<LRNParam>(attrs.parsed);
   if (SupportMKLDNN(inputs[0])) {
-    MKLDNNLRN_Forward(ctx, param, inputs[0], req[0], outputs[0]);
```
Comment (zheng-da) [inline src/operator/nn/lrn.cc]: it's better to keep req. ideally, we should use req in the forward function.

**Comment by zheng-da** [comment]
It looks good. Could you please rebase? I'll merge it.

**Comment by pengzhao-intel** [comment]
@zheng-da Done, please help merge.
